### ISSUE TITLE
New grafana panel plugin: business-table

### ISSUE THREAD

**Issue by Vonng** [Vonng/pigsty#452]
The volkovLabs stack now has a new grafana panel available:
- https://github.com/volkovlabs/business-table

But not publish yet

**Comment by Vonng** [comment]
```yaml

  - volkovlabs-echarts-panel
  - volkovlabs-image-panel
  - volkovlabs-form-panel
  - volkovlabs-variable-panel
  - volkovlabs-table-panel
  - knightss27-weathermap-panel
  - marcusolsson-dynamictext-panel
  - marcusolsson-treemap-panel
  - marcusolsson-calendar-panel
  - marcusolsson-hourly-heatmap-panel
  - marcusolsson-static-datasource
  - marcusolsson-json-datasource
  - volkovlabs-rss-datasource
  - volkovlabs-grapi-datasource

```

While currently this command fails:

```bash
grafana-cli plugins install volkovlabs-table-panel
```

We have to wait until it is available

**Comment by Vonng** [comment]
resolved and released in v3.0.3
